apps/miiverse-api/src/types/mongoose/post.ts

```diff
@@ -15,7 +15,9 @@ export interface IPost {
 
 	painting?: string; // base64, can be empty or undefined
 	screenshot?: string; // url fragment (leading /), can be empty or undefined
+	screenshot_thumb?: string; // url fragment (leading /), can be empty or undefined
 	screenshot_length?: number;
+	screenshot_aspect?: string; // '4:3' '5:4' '16:9'
```

Comment:
Miiverse-API also has some image processing in `services/api/routes/posts.ts` and `services/api/routes/communities.ts`. Should we also include the changes from the PR there?